### ISSUE TITLE
Clarify the timing of modifiers to Firepower for shooting

### ISSUE THREAD

**Issue by digi-brain** [GameBrains/er-core#5]
I think it really isn't very clear at the moment. And if I recall correctly, on a close read of the source texts, there was something that was fundamentally ambiguous or contradictory in the Epic 40,000 source texts. I didn't resolve this on my first pass because it requires a rules decision one way or another that was not clear in the original.

We should be able to make clear exactly when to halve or otherwise modify Firepower: before/after deduction for Blast markers; before/after placement of Blast markers; any other oddities.

Once we clarify this in the shooting rules then we shouldn't need to say anything about the timing elsewhere (eg, Ravenwing/Dodge).

**Comment by IJW-Wartrader** [comment]
From memory, the only ambiguity was whether you halved Firepower at the level of individual units or at the level of target detachments (aka after adding up Firepower), but doing it at the unit level has so many bizarre effects that I'm not aware of anyone suggesting playing it that way. A lot of the time, halving it at the unit level and rounding up wouldn't actually affect the Firepower total, and also contradicts _some_ of the Q&As.

A related issue was the sequence for working out Blast markers inflicted, halving Firepower, and reducing Firepower due to Blast markers, but that was an issue of players _assuming_ that inflicting BMs was based on the final Firepower instead of going back and reading the rules.

As per https://thehobby.zone/resources/e40k-compendium/Content/CoreRules/WorkOutBlastMarkers.htm you work out how many BMs are being inflicted 'before modification (for Assault orders, Blast markers on the detachment, etc)'.

So the full timing would be:

S1. Add up shooting Firepower - added up per target detachment.
S2. Work out Blast markers - worked out per target detachment.
S3. Work out how many hit dice to roll
1. Reduce Firepower by the number of Blast markers, if you're splitting fire you choose how to apply the reduction.
2. Halve Firepower for Assault or Dodge orders, war engine critical results etc. (Side note - for consistency this would also be the time to roll to lock on with SHWs.)
3. Take this final Firepower value to the firepower chart to work out how many dice to roll.

**Comment by IJW-Wartrader** [comment]
Expanding the side note about super-heavy weapons needing to lock on, because it's relevant here, plus a couple of slightly-related points...


### Lock on

Although the Shooting phase doesn't currently mention the need to lock on with SHWs, I think it's worth adding a note or calling it out in S3, and saying that it applies whenever that unit's or detachment's Firepower is halved. Then use similar text (or a partial?) on the main Super-heavy weapons page.

This will make the Critical Damage tables slightly simpler as they won't need to spell out the need to lock on, the overall rules text is more consistent, and it would cover future edge-cases like a detachment that has the Dodge ability and can include units with SHWs.


### Shooting with super-heavy weapons

In super-heavy-weapons.adoc > Shooting with super-heavy weapons, you've got a TODO note asking about how locking on works if not using war engine orders. This isn't relevant for two reasons - all sources of halving Firepower that I can think of also require SHWs to lock on, so it's not linked just to Assault orders, and secondly there are a lot of regular units that carry an SHW! ;)


### War engines, suppression, splitting fire, and locking on

This point is more speculative, and I need to bring it up in the FB groups. There are a bunch of FAQs about how Blast markers and SHWs work, text in the war engine rules for not splitting fire from one battery, and one of the Armageddon intro articles where Jervis tears his hair out talking about how misunderstood this all was, and how Blast markers applying per model fixed all this for Armageddon. A case in point being the three FAQs for Pulsars, all repeating that it's a single SHW for inflicting Blast markers, for being shut down by Blast markers, and for firefights.

Because...

Within the Epic 40k rules, as far as I can see, there was a concept of a 'war engine weapon mount' which was the smallest sub-division of firing for war engines. But this was never spelled out.

**Proposal** - all war engines have a number of weapon mounts, on data sheets this is each separate weapon line. for example a Warlord has four (left arm, right arm, left carapace, right carapace), while a Baneblade has one (turret guns), and a Shadowsword has two (Death Ray, sponson turrets).

A weapon mount:
- Cannot split fire between target detachments.
- Is a single weapon for criticals that destroy a weapon or limit the war engine to firing one weapon.

In addition, a weapon mount that does not have a normal Firepower value* counts as a single super-heavy weapon for the following purposes:
- Being suppressed by Blast markers.
- Inflicting Blast markers.
- Rolling to lock on.
- Adding up Firepower in a firefight.

*So everything that isn't a heavy weapon battery or the Firepower part of a Close Combat Weapon.

**Comment by IJW-Wartrader** [comment]
P.S. I'm aware that the proposal doesn't follow the style guide in multiple ways, but it's just the sketch version...

**Comment by digi-brain** [comment]
Haven't had time to read through this yet, but I'll get a look soon.

For reference in case it differs from what you have above, from the bottom of the thread (in relation to point no.2, ignore the red herring of point no.1): (https://www.facebook.com/groups/1952676958177757/permalink/3234394693339304):

>I think that the process would be more logical if it were like this:
>1. Count up the 'normal' Firepower for every shooting unit to get an initial running total.
>2. Reduce the running total by 1 for each Blast marker that is on the shooting unit.
>3. Halve the running total if the detachment is on Assault orders.
>4. Now you have the final total. Use this total both to find how many Blast markers to place on the target unit and to find how many hit dice to roll.
> 
>This is a bit different to the rules as written and avoids the situation in which detachments on Assault orders retain some 'undiminished' shooting benefits. Something to think about for E40k Remastered perhaps?

**Comment by IJW-Wartrader** [comment]
I'm hesitant about simply reversing the sequence, but let's see if we get much feedback via the two polls I've just put up:

https://www.facebook.com/groups/1004227353047038/permalink/2260319480771146/
https://www.facebook.com/groups/1952676958177757/permalink/3777141742397927/

**Comment by digi-brain** [comment]
Yeah, it's a bit tricky for us to decide where we should actually change the rules. But using the effective Firepower does seem more logical and intuitive, and like the sort of thing that many people might do without realising it was 'wrong' anyway.
